packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,33 +1,70 @@
 // ----------------
-// Variables
+// 1.  CSS Variables
+// 2.  Dark Mode
+// 3.  Autocomplete
+// 4.  Panel
+// 5.  Sources
+// 6.  Hit Layout
+// 7.  Panel Header
+// 8.  Panel Footer
+// 9.  Detached Mode
+// 10. Gradients
+// ----------------
+
+// Note:
+// This theme is trying to reflect the markup of autocomplete with SCSS indentation.
+// We use the SASS `@at-root` function to remove unecessary specificities.
+
+// ----------------
+// 1. CSS Variables
 // ----------------
 :root {
```

Comment:
should these variables be set on the autocomplete root instead of the document root?

Reply:
true, I think putting it on .aa-Autocomplete means it is much less likely to conflict with other authored styles

Reply:
Problem is that we have several wrapper at root for autocomplete. The detached mode spawn at body level. Means that we would have a multi selector class. I'm not against it, I thought it was just 'simpler' like this.

Reply:
Indeed, the detached mode is in `aa-DetachedOverlay` not `aa-Autocomplete`.

Again, since variables are prefixed, `:root` should be fine.